src/scripts/main.js

```diff
@@ -2,7 +2,17 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('tbody');
 
-// write your code here
+people.forEach(person => {
+  const personRow = `<tr>
+  <td>${person.name}</td>
+  <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
+  <td>${person.born}</td>
+  <td>${person.died}</td>
+  <td>${person.age = person.died - person.born}</td>
+  <td>${person.century = Math.ceil(person.died / 100)}</td>
+</tr>`;
+
+  tableBody.innerHTML += personRow;
```

Comment:
Appending rows using 'innerHTML' may lead to performance issues. Instead, create table rows and cells using 'document.createElement', set their contents and append them to the table using 'appendChild'.